media/js/base/mozilla-lazy-load.js

```diff
@@ -73,16 +73,18 @@ if (typeof window.Mozilla === 'undefined') {
      * @param _selector (String) - CSS selector for target images e.g. '.list-item img'.
      */
     LazyLoad.loadAllFallback = function(_selector) {
-        $(_selector).each(function() {
-            var srcset = this.getAttribute('data-srcset');
+        document.querySelectorAll(_selector)
+            .forEach(function(currentValue) {
```

Comment:
`forEach` on a NodeList isn't supported on Internet Explorer, so we may need to use a traditional `for` loop here.

Reply:
Replaced this `forEach` with a normal `for` loop